xgettext.js

```diff
@@ -164,7 +164,9 @@ XGettext.prototype._discoverMatches = function( parsedInput ) {
 
 	traverse( parsedInput.ast, {
 		CallExpression: function( node ) {
-			var functionName = ( node.callee.property ) ? node.callee.property.name : node.callee.name;
+			// Pull the resultingFunction out of (0, resultingFunction)()
+			var callee = ( node.callee.type === 'SequenceExpression' ) ? _.last( node.callee.expressions ) : node.callee;
```

Comment:
Also, it's technically possible for recursion here, e.g.:

```js

it( 'should match functions that are the last element of a recursive sequence (comma) expression', function() {
	var source = '(0, (0, transpilerGeneratedName._))("Hello World!")',
		matches = new XGettext().getMatches( source );

	expect( matches ).to.deep.equal( [{ 'string': 'Hello World!', line: 1 }] );
});
```

Do we care to support this?